src/live_restore/live_restore_fs.c

```diff
@@ -1394,8 +1394,9 @@ __wt_live_restore_clean_metadata_string(WT_SESSION_IMPL *session, char *value)
     WT_CONFIG_ITEM v;
     WT_DECL_RET;
 
-    /* FIXME-WT-14231 Allow taking backups during live restore in the COMPLETE phase. */
-    WT_ASSERT_ALWAYS(session, !F_ISSET(S2C(session), WT_CONN_LIVE_RESTORE_FS),
+    WT_ASSERT_ALWAYS(session,
```

Comment:
Instead of crashing out we should return EBUSY, the user wouldn't expect the process to crash. We could leave a diagnostic assert in for our own testing reasons.